Move SHOW SETTINGS to ops manual (#3033)
neo4j/docs-operations: modules/ROOT/pages/configuration/index.adoc

* xref:configuration/dynamic-settings.adoc[Update dynamic settings] -- How to configure certain Neo4j parameters while Neo4j is running.
* xref:configuration/migrate-configuration.adoc[Migrate configurations] -- How to migrate configuration settings from a previous version of Neo4j to a new version.
* xref:configuration/validate-config.adoc[Validate configurations] -- How to validate Neo4j and Log4j configurations.

> * xref:configuration/list-settings.adoc[List settings] -- How to list the settings available in a database using the `SHOW SETTINGS` command.

Reviewer: ```suggestion
* xref:configuration/list-settings.adoc[Show configuration settings] -- How to list all available configuration settings using the `SHOW SETTINGS` command.
```
